internal/hmyapi/apiv1/blockchain.go

```diff
@@ -261,8 +263,10 @@ func (s *PublicBlockChainAPI) GetBlockSigners(ctx context.Context, blockNr rpc.B
 		if err != nil {
 			return nil, err
 		}
-		blsPublicKey := new(bls.PublicKey)
-		validator.BLSPublicKey.ToLibBLSPublicKey(blsPublicKey)
+		blsPublicKey := new(bls_core.PublicKey)
```

Comment:
these allocations are not necessary right?

Reply:
it is needed if you want to inline the error check. It shouldn't have performance difference as anyway a new pointer needs to be allocated. So I will leave as is.